frontend/src/components/maps/FindMorePropertiesButton.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Button, OverlayTrigger, Popover } from 'react-bootstrap';
+import { BsXSquareFill } from 'react-icons/bs';
+import { FaSign } from 'react-icons/fa';
+import styled from 'styled-components';
+import FindMorePropertiesForm from 'components/SearchBar/FindMorePropertiesForm';
+
+const ButtonContent = styled.div`
+  display: flex;
+  .p {
+    font-size: 16px;
+  }
+`;
+
+const StyledPopover = styled(Popover)`
+  max-width: 100%;
+`;
+
+const CloseButton = styled(BsXSquareFill)`
+  margin-left: 256px;
+  cursor: pointer;
+`;
+
+const TitleForSaleSign = styled(FaSign)`
+  fill: #003366;
+  margin-right: 5px;
+`;
+
+/** this component contains the trigger for additional filter options off the base filter */
+export const FindMorePropertiesButton: React.FC<any> = () => {
+  const TitleContent = () => (
+    <div style={{ display: 'flex' }}>
+      <TitleForSaleSign size={42} />
+      <h3 style={{ color: '#003366' }}>Find available surplus properties</h3>
+      <CloseButton onClick={() => document.body.click()} />
+    </div>
+  );
+
+  /** this provides a way to create a form with tooltip like behaviour in the overlay trigger */
+  const popover = (
+    <StyledPopover id="popover-basic">
+      <Popover.Title>
+        {' '}
+        <TitleContent />{' '}
+      </Popover.Title>
+      <Popover.Content>
+        <FindMorePropertiesForm />
+      </Popover.Content>
+    </StyledPopover>
+  );
+
+  return (
+    <OverlayTrigger trigger="click" rootClose transition overlay={popover} placement="bottom">
```

Comment:
Right now it closes via the 'x' button and clicking outside of the form. But I have a feeling this isn't desired behaviour. Currently adjusting, going to confirm with Bobbi.